spec/support/rspec_playwright.rb

```diff
@@ -29,6 +29,12 @@ def self.headless
     end
   end
 
+  # rubocop:disable Naming/MethodName
+  def self.slowMo
+    ENV.fetch('SLOWMO', 0).to_i
+  end
+  # rubocop:enable Naming/MethodName
```

Comment:
Then there's no need to disable rubocop: 
```suggestion
  def self.slow_mo
    ENV.fetch('SLOWMO', 0).to_i
  end
```